lib/tapioca/cli.rb

```diff
@@ -82,7 +82,7 @@ def todo
     option :workers,
       aliases: ["-w"],
       type: :numeric,
-      default: nil,
+      default: 1,
```

Comment:
From https://github.com/Shopify/tapioca/pull/566, it seems this isn't the right way to fix the default configuration values?

Reply:
Removed the default CLI option according to that PR.